src/config.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
   'secret': 'calvinandjeff',
+  // Having the database here will cause some grief when you push to Heroku.
+  // Would be better to push this back out to srcServer.js. - Harold
   'database': 'mongodb://localhost/giphys'
 };
```

Comment:
this hasn't been in our code for a long time.  not sure how it showed up here.  anyway, taken care of